spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/TestBaseWithCatalog.java

```diff
@@ -73,7 +73,7 @@ public static void dropWarehouse() throws IOException {
     }
   }
 
-  @TempDir protected File temp;
+  @TempDir protected java.nio.file.Path temp;
```

Comment:
Super nit, but while we're doing this: I find it easier to read if we don't use the fully qualified Path name here, rather use the fullly qualified name for the hadoop `Path` in `dropWarehouse`.